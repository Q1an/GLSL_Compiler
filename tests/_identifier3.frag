_id1
__id2
__id345_
__id34dskf_34
