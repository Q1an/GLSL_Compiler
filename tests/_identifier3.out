_id1         line 1 cols 1-4 is T_Identifier 
__id2        line 2 cols 1-5 is T_Identifier 
__id345_     line 3 cols 1-8 is T_Identifier 
__id34dskf_34 line 4 cols 1-13 is T_Identifier 
